src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -111,7 +111,10 @@ export class PastAppointmentsList extends React.Component {
 
     return (
       <div role="tabpanel" aria-labelledby="tabpast" id="tabpanelpast">
-        <h3>Past appointments</h3>
+        <h3 tabIndex="-1" id="pastAppts">
+          Past appointments
+          <span class="sr-only"> past three months</span>
```

Comment:
@jbalboni Agreed, this work seems tied to the update button. The SR-only text will need to change when users press the Update button, so that's the right time to change labels also.

Reply:
That makes sense. I'll remove the extra screen reader text.